### ISSUE TITLE
screen reader optimized status bar item is missing in aux window

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#35252]
Testing -199023

![Screenshot 2023-11-28 at 1 08 10 PM](https://github-com/microsoft/vscode/assets/29464607/39671283-22bd-4228-8247-d6cc26a2b803)
